automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/Leonardo.scala

```diff
@@ -174,6 +174,12 @@ object Leonardo extends RestClient with LazyLogging {
       mapper.readValue(response, classOf[ContentItem])
     }
 
+    def proxyBasePath(googleProject: GoogleProject, clusterName: ClusterName): String =
+      s"proxy/${googleProject.value}/${clusterName.string}"
+
+    def jupyterProxyBasePath(googleProject: GoogleProject, clusterName: ClusterName): String =
```

Comment:
i've decided to not use the new paths quite yet in the tests. all of our apps still use the legacy paths so it felt wrong to leave those uncovered. the alternative was to test both paths but then we would have tests split or duplicated. 

i think in planning for the deprecation and removal of the legacy paths, we can figure out a better plan for these tests.

i'm thinking something like

1. email app teams about removal of the legacy path in X days. tell them it will become unsupported prior to removal
2. move tests to use new path
3. remove old path

...or something similar